healthcheck/healthcheck_backend.go

```diff
@@ -0,0 +1,27 @@
+package healthcheck
+
+import (
+	_ "github.com/lib/pq"
+	"github.com/twitchscience/rs_ingester/loadclient"
+	"github.com/twitchscience/rs_ingester/metadata"
+	"github.com/twitchscience/scoop_protocol/scoop_protocol"
+)
+
+type HealthCheckBackend struct {
+	postgresScoopConnection loadclient.Loader
```

Comment:
postgres is so frequently abbreviated to pg that it would be safe to do here:

pgScoopConn
pgBackend